model/lib/mailbox/presentation_mailbox.dart

```diff
@@ -89,6 +89,8 @@ class PresentationMailbox with EquatableMixin {
 
   bool get isOutbox => name == lowerCaseOutboxMailboxName || role == roleOutbox;
 
+  bool get isSubscribedMailbox => isSubscribed == null || isSubscribed?.value == true;
```

Comment:
please take care. How about?
```suggestion
  bool get isSubscribedMailbox => isSubscribed != null && isSubscribed?.value == true;
```

Reply:
it so okay